lib/util.js

```diff
@@ -29,6 +29,20 @@ function getUserId(userId) {
   return userId
 }
 
+function getUserProperties(userPropertiesAsString) {
+  let result = {}
+  if (!userPropertiesAsString) {
+    return result
```

Comment:
return null

Reply:
I don't like returning `null`, I prefer `{}` as 0-value

Reply:
The reason is that function 
```js
function getGroups(groups) {
  return groups.split(',').filter(longerThan(0))
}
```
which is used to retrieve the list of user's groups, returns an empty list as 0-value.